spring-boot-security-saml/pom.xml

```diff
@@ -35,7 +35,12 @@
         <dependency>
             <groupId>org.springframework.security.extensions</groupId>
             <artifactId>spring-security-saml2-core</artifactId>
-            <version>1.0.2.RELEASE</version>
+            <version>1.0.4.RELEASE</version>
+        </dependency>
+        <dependency>
```

Comment:
why do we need this?

Reply:
This is the latest spring-security-saml library. Unless I don't undestand the question.

Reply:
I was talking about the json dependency

Reply:
Ohh, The github showed me the saml dependency. OK. I was fixing the build and forgot that this dependency has been repacjed under the spring ubmrella. I'll remove it.

Reply:
Done